.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
What is DS_Store?

Reply:
a file osx puts into a directory

Reply:
Would it make sense for people to add .DS_Store to their global git configs, rather than adding them to per-repo .gitignore files? I imagine .DS_Store should be ignored in all repos.

Reply:
~done ✅